test/features/kogito_service_performance.feature

```diff
@@ -148,13 +149,13 @@ Feature: Kogito Service Performance
       }
       """
 
-    Then HTTP GET request on service "process-springboot-example" with path "orders" should return an array of size <requests> within 1 minutes
-    And HTTP GET request on service "process-springboot-example" with path "orderItems" should return an array of size <requests> within 1 minutes
+    #Then HTTP GET request on service "process-springboot-example" with path "orders" should return an array of size <requests> within 1 minutes
```

Comment:
Same as above.

Reply:
This is because of issues which are listed on the top of the feature file as we currently don't have pagination and it is giving 504 Gateway time-out for 320k processes.